ethexe/processor/src/handling/events.rs

```diff
@@ -105,12 +105,16 @@ impl ProcessingHandler {
                 value,
             } => {
                 self.update_state(actor_id, |state, storage, transitions| -> Result<()> {
-                    let Some(ValueWithExpiry {
-                        value: claimed_value,
-                        expiry,
-                    }) = state
-                        .mailbox_hash
-                        .modify_mailbox(storage, |mailbox| mailbox.remove(source, replied_to))
+                    let Some(MailboxMessage {
+                        value:
+                            ValueWithExpiry {
```

Comment:
Value with expiry now could be deprecated: expiry relates to message itself, so MailboxMessage {
payload,
value,
expiry,
kind ... 
}